src/NUnitFramework/framework/Constraints/MsgUtils.cs

```diff
@@ -161,7 +166,7 @@ public static string FormatValue(object val)
         /// <param name="collection">The collection containing elements to write.</param>
         /// <param name="start">The starting point of the elements to write</param>
         /// <param name="max">The maximum number of elements to write</param>
-        public static string FormatCollection(IEnumerable collection, long start, int max)
+        public static string FormatCollection(IEnumerable collection, long start = 0, int max = DefaultMaxItems)
```

Comment:
Decided to add default parameter, since it is used in most cases with those arguments.
Let me know if there are any objections.

Reply:
Personally, I really like this. It makes it easy to ensure future calls to this will always respect the defaults unless the author explicitly decides not to.
Is there any way you could expose just this change in a separate commit on your branch? I'd like to graft it onto my own branch for https://github.com/nunit/nunit/pull/3502 to help simplify things there

Reply:
Updated PR to have separate commit for MsgUtils change. Still, not sure it's worth reusing it until this change is approved.

Reply:
I really like this change! Can I suggest updating all the places this method is currently used to make use of it too? The vast majority are currently passing in zero and ten.

Reply:
Done!